docker/run_mailtck.sh

```diff
@@ -22,10 +22,23 @@ else
   exit 1
 fi
 
-if [[ "$JDK" == "JDK11" || "$JDK" == "jdk11" ]];then
-  export JAVA_HOME=${JDK11_HOME}
-  export PATH=$JAVA_HOME/bin:$PATH
-fi
+export JAVA_HOME=${JDK11_HOME}
+
+if [[ "$JDK" == "JDK12" || "$JDK" == "jdk12" ]];then
+  export JAVA_HOME=${JDK12_HOME}
+elif [[ "$JDK" == "JDK13" || "$JDK" == "jdk13" ]];then
+  export JAVA_HOME=${JDK13_HOME}
+elif [[ "$JDK" == "JDK14" || "$JDK" == "jdk14" ]];then
+  export JAVA_HOME=${JDK14_HOME}
+elif [[ "$JDK" == "JDK15" || "$JDK" == "jdk15" ]];then
+  export JAVA_HOME=${JDK15_HOME}
+elif [[ "$JDK" == "JDK16" || "$JDK" == "jdk16" ]];then
+  export JAVA_HOME=${JDK16_HOME}
+elif [[ "$JDK" == "JDK17" || "$JDK" == "jdk17" ]];then
+  export JAVA_HOME=${JDK17_HOME}
+fi  
```

Comment:
would it be possible to make this open-ended? I mean we probably do not want to edit this file every time there is a new JDK available

Reply:
we planning to use JDK from Jenkins jobs, to reduce size of docker images and to avoid run script changes.
Post addition of JDK from Jenkins jobs, we will make this script open ended.